### DISCUSSION TITLE
[next/image] - How to show protected image with HttpOnly token?

### DISCUSSION THREAD

**Discussion by vendramini** [vercel/next.js#19497] [Help]
## Describe the bug

I don't know if it is a bug, but the situation is: I've some images which are protected from public, only some users can access them. When using the `<Image/>` component to get them the headers aren't been sent. I'm using `HttpOnly` to store my token. In that way, the `<Image/>` component doesn't has permission to get the image.

Currently I've fixed that using the HTML <img/> tag, but I would like to use next's Image component.

## System information

- OS: Linux Mint
- Version of Next.js: 10.0.2
- Version of Node.js: 12
- Deployment: yarn dev = node server.js (custom server.js)

**Comment by brobin002** [comment]
Late response but I found a solution. Just pass the token in request's header or cookie and set up a middleware to check the token. If someone is interested I can provide an example using nodejs. But this is not related to Next.
